docs/development_and_contributing.rst

```diff
@@ -32,10 +32,16 @@ directory:
 .. code-block:: bash
 
   pyenv install 3.9.x
-  pyenv install 3.8.x
+  pyenv install 3.10.x
```

Comment:
it's best to develop on the lowest version. Let's keep this concise and remove the additions here

Reply:
Done, sorry it took a bit.